build.gradle.kts

```
@@ -131,6 +136,14 @@ tasks.test {
   }
 }
 
+tasks.distZip { dependsOn(tasks.shadowJar) }
+
+tasks.distTar { dependsOn(tasks.shadowJar) }
+
+tasks.named("startScripts") { dependsOn(tasks.shadowJar) }
+
+tasks.named("startShadowScripts") { dependsOn(tasks.jar) }
```

Had to add these lines or `./gradlew build` wouldn't work for me.

---

Shouldn't this one be:

```suggestion
tasks.named("startShadowScripts") { dependsOn(tasks.shadowJar) }
```

---

I added the exact dependencies that gradle complained should be there. You can try to reproduce it locally with `./gradlew build`. @holzensp had the same error also.

---

Some of these come from the application plugin, which we can probably just get rid of, since we're not using it. But that can be in another PR.